src/test/java/net/sf/jabref/importer/fileformat/InspecImportTest.java

```diff
@@ -0,0 +1,109 @@
+package net.sf.jabref.importer.fileformat;
+
+import static org.junit.Assert.*;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import net.sf.jabref.model.entry.BibtexEntryTypes;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import org.junit.Before;
+import org.junit.Test;
+
+public class InspecImportTest {
+
+    private InspecImporter inspecImp;
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        this.inspecImp = new InspecImporter();
+    }
+
+    @Test
+    public void testIsRecognizedFormatAccept() throws IOException {
+        List<String> testList = Arrays.asList("InspecImportTest.txt", "InspecImportTest2.txt");
+        for (String str : testList) {
+            try (InputStream inStream = InspecImportTest.class.getResourceAsStream(str)) {
+                assertTrue(inspecImp.isRecognizedFormat(inStream));
+            }
+        }
+    }
+
+    @Test
+    public void testIsRecognizedFormatReject() throws IOException {
+        List<String> testList = Arrays.asList("CopacImporterTest1.txt", "CopacImporterTest2.txt",
+                "IEEEImport1.txt", "IsiImporterTest1.isi", "IsiImporterTestInspec.isi", "IsiImporterTestWOS.isi",
+                "IsiImporterTestMedline.isi", "RisImporterTest1.ris", "InspecImportTestFalse.txt");
+
+        for (String str : testList) {
+            try (InputStream inStream = InspecImportTest.class.getResourceAsStream(str)) {
+                assertFalse(inspecImp.isRecognizedFormat(inStream));
+            }
+        }
+    }
+
+    @Test
+    public void testCompleteBibtexEntryOnJournalPaperImport() throws IOException {
+        try (InputStream inStream = InspecImportTest.class.getResourceAsStream("InspecImportTest2.txt")) {
+            List<BibEntry> entries = inspecImp.importEntries(inStream, new OutputPrinterToNull());
+            assertEquals(1, entries.size());
+            BibEntry entry = entries.get(0);
+
+            assertEquals("The SIS project : software reuse with a natural language approach", entry.getField("title"));
+            assertEquals("Prechelt, Lutz", entry.getField("author"));
+            assertEquals("1992", entry.getField("year"));
+            assertEquals("Abstrakt", entry.getField("abstract"));
+            assertEquals("key", entry.getField("keywords"));
+            assertEquals("10000", entry.getField("journal"));
+            assertEquals("20", entry.getField("pages"));
+            assertEquals("19", entry.getField("volume"));
+            assertEquals(BibtexEntryTypes.ARTICLE.getName().toLowerCase(), entry.getType());
```

Comment:
Yes instead of 
```
assertEqual("bla", entry.getField("field"));
assertEqual("foo", entry.getField("field2"));
```
you now write
```
BibEntry expected = new Entry();
expected.setField("field1", "bla")
expected.setField("field2", "foo")
BibtexEntryAssert.assertEqual(expected, entry);
```